activesupport/test/core_ext/range_ext_test.rb

```diff
@@ -136,6 +140,10 @@ def test_should_include_identical_exclusive_with_floats
     assert((1.0...10.0).include?(1.0...10.0))
   end
 
+  def test_include_returns_false_for_backwards
+    assert_not((1..10).include?(5..3))
+  end
```

Comment:
Could you move this to line 63? That way all three tests are below the existing `...other_with_exclusive_end` test for their operator.